recipes/libpqxx/all/conanfile.py

```diff
@@ -67,6 +67,9 @@ def _configure_cmake(self):
             self._cmake = CMake(self)
             self._cmake.definitions["BUILD_DOC"] = False
             self._cmake.definitions["BUILD_TEST"] = False
+            if Version(self.version) >= "7.3.1":
+                self._cmake.definitions["INSTALL_TEST"] = False
```

Comment:
Do we really need this?
I think if BUILD_TEST is False, then directory is not added, that's why tests are not built and not installed.

Reply:
Thanks for the review.

As you pointed out, test is not built, so this option does not affect the build result.
I decided that it would be easier for everyone to understand that it will not be installed by explicitly specifying it.
The `BUILD_TEST` (by default `False`) is also specified with the same idea.

Should I remove it?

Reply:
Yes, I would suggest removing this option, it makes the recipe a bit more standard and easier.

Reply:
I need to mention that I'm not the maintainer of CCI, just random contributor, so you might want to hear other opinion as well.

Reply:
Thanks for your opinion.
I'll wait for other reviewers' opinions.

Reply:
Please remove it, the extra complexity is not required.